@ecomplus/storefront-components/src/js/TheAccount.js

```diff
@@ -52,7 +52,8 @@ export default {
 
   data () {
     return {
-      favoriteIds: []
+      favoriteIds: [],
+      navTabs: [i19registration, i19orders, i19favorites]
```

Comment:
Isso funcionou? 🙄 
`this.i19registration` (por exemplo) é uma string porque pega o computed aqui: https://github.com/ecomplus/storefront/blob/cad71addad23aed00c55001bb29f6d874e024965/%40ecomplus/storefront-components/src/js/TheAccount.js#L69
`i19registration` sem o this _pode_ (dependendo do alias do `@ecomplus/i18n` no bundler) ser um objeto porque pega o named import daqui https://github.com/ecomplus/storefront/blob/cad71addad23aed00c55001bb29f6d874e024965/%40ecomplus/storefront-components/src/js/TheAccount.js#L9
São diferentes...

Reply:
Se você usar a constante aí você **não vai usar os respectivos computeds, portanto deve removê-los lá**, fora isso nesse caso você teria que usar a função `i18n`:

```suggestion
      navTabs: [i18n(i19registration), i18n(i19orders), i18n(i19favorites)]
```

Reply:
Entendi... eu tinha testado com o this.i19registration no data, como vc sugeriu, mas dessa forma não funcionou. Só funcionou certinho declarando o navTabs como vazio na data e depois adicionando os elementos no created.
Do jeito que eu fiz funcionou tbm, mas não tinha pensado nessa possibilidade de acabar chamando o objeto em vez da string. Arriscado mesmo... 😬